Kronos-Shifts-Connector/Microsoft.Teams.Shifts.Integration/Microsoft.Teams.Shifts.Integration.API/Controllers/OpenShiftController.cs

```diff
@@ -103,9 +110,9 @@ public async Task<ShiftsIntegResponse> CreateOpenShiftFromTeamsAsync(Models.Inte
 
             var openShiftDetails = new
             {
-                KronosStartDateTime = this.utility.UTCToKronosTimeZone((DateTime)(openShift.DraftOpenShift?.StartDateTime ?? openShift.SharedOpenShift?.StartDateTime), team.KronosTimeZone),
-                KronosEndDateTime = this.utility.UTCToKronosTimeZone((DateTime)(openShift.DraftOpenShift?.EndDateTime ?? openShift.SharedOpenShift?.EndDateTime), team.KronosTimeZone),
-                DisplayName = openShift.DraftOpenShift?.DisplayName ?? openShift.SharedOpenShift?.DisplayName ?? null,
+                KronosStartDateTime = this.utility.UTCToKronosTimeZone((DateTime)openShift.SharedOpenShift?.StartDateTime, team.KronosTimeZone),

```

Comment:
I would suggest that if SharedOpenShift is null then you should return an error and therefore you should not need SharedOpenShift?.XYZ